op-node/rollup/derive/calldata_source.go

```diff
@@ -41,6 +41,12 @@ func (ds *DataSourceFactory) OpenData(ctx context.Context, id eth.BlockID, batch
 	return NewDataSource(ctx, ds.log, ds.cfg, ds.fetcher, id, batcherAddr)
 }
 
+// minimalDataSourceConfig regroups the mandatory rollup.Config fields needed for DataFromEVMTransactions.
+type minimalDataSourceConfig struct {
```

Comment:
i have a slight personal nit on the name because i think its verbose, could do without the minimal prefix

Reply:
Fixed.